new-lamassu-admin/src/styling/theme.js

```diff
@@ -41,121 +37,221 @@ export default createTheme({
       default: backgroundColor
     }
   },
-  overrides: {
-    MuiRadio: {
-      colorSecondary: {
-        color: secondaryColor
+})
+
+theme = createTheme(theme, {
```

Comment:
If I understood [the docs](https://mui.com/material-ui/customization/theming/#createtheme-options-args-theme) correctly, this reassignment could be avoided by simply removing these three lines?

Reply:
The problem here is using theme configuration in the object that I'm re-assigning. I can't reference the theme unless I create a theme first, this is exactly like the docs suggest handling it under "theme composition"